app/src/ui/history/commit-list.tsx

```diff
@@ -8,26 +8,52 @@ import { IGitHubUser } from '../../lib/databases'
 const RowHeight = 50
 
 interface ICommitListProps {
-  readonly onCommitChanged: (commit: Commit) => void
-  readonly onScroll: (start: number, end: number) => void
-  readonly onRevertCommit: (commit: Commit) => void
-  readonly onViewCommitOnGitHub: (sha: string) => void
-  readonly repository: Repository
-  readonly history: ReadonlyArray<string>
-  readonly commits: Map<string, Commit>
+  /** The GitHub repository associated with this commit (if found) */
+  readonly gitHubRepository: GitHubRepository | null
+
+  /** The list of commits to display, in order. */
+  readonly commits: ReadonlyArray<string>
+
+  /** The commits loaded, keyed by their full SHA. */
+  readonly commitLookup: Map<string, Commit>
```

Comment:
Previously this was `commits`, and if you didn't look at the type signature you might think that this was the source for the list. So by adding the `Lookup` suffix I hope this becomes a bit clearer.